### ISSUE TITLE
tailwind breaks rails mailer generator

### ISSUE THREAD

**Issue by rubys** [rails/tailwindcss-rails#90]
With Rails 7.0.0.alpha2, running

```
rails new foo --css tailwind
cd foo
bin/rails generate mailer Order received shipped
```

Produces:

```
      create  app/mailers/order_mailer.rb
       error  tailwindcss [not found]
      invoke  test_unit
      create    test/mailers/order_mailer_test.rb
      create    test/mailers/previews/order_mailer_preview.rb
```

What's missing:

```
      invoke  erb
      create    app/views/order_mailer
      create    app/views/order_mailer/received.text.erb
      create    app/views/order_mailer/received.html.erb
      create    app/views/order_mailer/shipped.text.erb
      create    app/views/order_mailer/shipped.html.erb
```

**Comment by rubys** [comment]
Perhaps the fix is merely awaiting release?  https://github.com/rails/tailwindcss-rails/commit/c3177f4a067ac4e296966b5bcfaec1dad123d4df

**Comment by dhh** [comment]
Released 0.5.2 with this included 👍
